.gitignore

```diff
@@ -9,3 +9,6 @@ __pycache__/
 
 data.txt
 pass.txt
+
+SOARback/db/SOARback_interventions.db
+SOARback/db/SOARback_users.db
```

Comment:
A lot of my comments will just be for understanding purposes. So these will be the locations where the interventions and the user login information will be stored?

Reply:
yeah so sqlite makes little files to track the contents of the db, the SOARback_interventions.db and SOARback_users.db file is where it stores those files